sdk/communication/Azure.Communication.Chat/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Release History
 
 ## 1.0.0-beta.4 (Unreleased)
-
+- Support for CreateChatThreadResult and AddChatParticipantsResult to handle partial errors in batch calls.
```

Comment:
We should be updating the changelog according to the Azure Guidance and adding subtitles such as `Breaking Change` or `Added`
https://azure.github.io/azure-sdk/policies_releases.html#changelog-guidance

Reply:
sounds good @minnieliu . Will update the changelog in the next PR accordingly